staging/cert-manager-setup/templates/certificates.yaml

```diff
@@ -29,5 +32,21 @@ spec:
     - {{ . | quote -}}
   {{- end }}
 {{- end }}
+{{- if .ipAddresses }}
+  ipAddresses:
+  {{- range .ipAddresses }}
+    - {{ . | quote -}}
+  {{- end }}
+{{- end }}
+  keyAlgorithm: {{ .keyAlgorithm | default "rsa" }}
+  keyEncoding: {{ .keyEncoding | default "pkcs1" }}
+  keySize: {{ .keySize | default "2048" }}
+{{- if .organization }}
```

Comment:
Please add commented out example values to `values.yaml` file so they are discoverable and self-documenting:

This is example of existing `clusterissuers` property
```
clusterissuers: []
#  - name: kubernetes-ca
#    spec:
#      ca:
#        secretName: kubernetes-intermediate-ca
#  - name: my-ca
#    spec:
#      ca:
#        secretName: my-certificate-secret
```